nodejs/eks/package.json

```diff
@@ -13,7 +13,9 @@
         "format": "prettier --list-different --write .",
         "lint-check": "yarn run format-check && tslint -c ../tslint.json -p tsconfig.json",
         "lint": "yarn run format && tslint -c ../tslint.json -p tsconfig.json --fix",
-        "test": "jest"
+        "test": "jest",
+        "dedupe-deps": "yarn-deduplicate",
```

Comment:
We had some duplicate entries in the lock file, added those tools to dedupe them